nixos/modules/services/web-apps/healthchecks.nix

```
@@ -21,6 +21,7 @@ let
       sudo='exec /run/wrappers/bin/sudo -u ${cfg.user} --preserve-env --preserve-env=PYTHONPATH'
     fi
     export $(cat ${environmentFile} | xargs)
+    ${lib.optionals (cfg.settingsFile != null) "export $(cat ${cfg.settingsFile} | xargs)"}
```

`lib.optionals` expects list, use `lib.optionalString` instead.
```suggestion
    ${lib.optionalString (cfg.settingsFile != null) "export $(cat ${cfg.settingsFile} | xargs)"}
```

---

Fixed, lib.optionals was outputting the correct results so I didn't notice, Thanks.